curator-framework/src/main/java/org/apache/curator/framework/listen/MappingListenerManager.java

```diff
@@ -28,7 +28,7 @@
 import java.util.function.Function;
 
 /**
- * Upgraded version of {@link org.apache.curator.framework.listen.ListenerContainer} that
+ * Upgraded version of ListenerContainer that
  * doesn't leak Guava's internals and also supports mapping/wrapping of listeners
```

Comment:
```suggestion
 * Subclass of {@link ListenerManager} that supports mapping/wrapping of listeners.
```

Since `ListenerContainer` removed, I don't think we have to mention this history in the nightly document.